src/toolbar/toolbar-group.jsx

```diff
@@ -19,17 +18,20 @@ function getStyles(props, state) {
 
   const styles = {
     root: {
-      float,
       position: 'relative',
       marginLeft: firstChild ? -marginHorizontal : undefined,
       marginRight: lastChild ? -marginHorizontal : undefined,
+      display: 'flex',
+      justifyContent: 'space-between',
+      flexWrap: 'wrap',
```

Comment:
Yes. The wrap is enabled on the `ToolbarGroup` and not on the `Toolbar` on purpose. If the wrap is removed the child elements within the ToolbarGroup are not as flexible when the screen size becomes smaller.

I was trying to replicate the current toolbar behavior. For the one with the flex design the wrap is required for the toolbar to be flexible enough to hold elements when the screen becomes smaller without breaking.